src/commands/new.ts

```diff
@@ -62,63 +61,68 @@ export default {
     p(` █ Powered by ${red("Infinite Red")} - https://infinite.red`)
-    p(` █ Using ${cyan(cli)}`)
+    p(` █ Using ${cyan(expo ? "expo-cli" : "ignite-cli")}`)
     p(` ────────────────────────────────────────────────\n`)
 
-    startSpinner("Igniting app")
-
-    const summoningExpo = "Summoning Expo CLI"
-    // generate the project
-    await spawnProgress(log(cliString), {
-      env: cliEnv,
-      onProgress: (out: string) => {
-        out = log(out.toString())
-
-        stopSpinner("Igniting app", "🔥")
-
-        if (expo) {
-          if (out.includes("Using Yarn")) {
-            startSpinner(summoningExpo)
-          }
-          if (out.includes("project is ready")) {
-            stopSpinner(summoningExpo, "🪔")
-            startSpinner("Cleaning up Expo install")
-          }
-        } else {
-          if (out.includes("Welcome to React Native!")) {
-            startSpinner(" 3D-printing a new React Native app")
-          }
-          if (out.includes("Run instructions for")) {
-            stopSpinner(" 3D-printing a new React Native app", "🖨")
-            startSpinner("Cooling print nozzles")
+    if (expo) {
+      const expoCLIString = `npx expo-cli init ${projectName} --template ${boilerplatePath} --non-interactive`
+      log({ expoCLIString })
+
+      // generate the project
+      startSpinner("Igniting app")
+      await spawnProgress(log(expoCLIString), {
+        env: cliEnv,
+        onProgress: (out: string) => {
+          stopSpinner("Igniting app", "🔥")
+
+          out = log(out.toString())
+
+          if (expo) {
+            if (out.includes("Using Yarn")) {
+              startSpinner("Summoning Expo CLI")
+            }
+            if (out.includes("project is ready")) {
+              stopSpinner("Summoning Expo CLI", "🪔")
+              startSpinner("Cleaning up Expo install")
+            }
+          } else {
+            if (out.includes("Welcome to React Native!")) {
+              startSpinner(" 3D-printing a new React Native app")
+            }
+            if (out.includes("Run instructions for")) {
+              stopSpinner(" 3D-printing a new React Native app", "🖨")
+              startSpinner("Cooling print nozzles")
+            }
           }
-        }
-      },
-    })
+        },
+      })
 
-    if (expo) {
-      stopSpinner(summoningExpo, "🪔")
+      stopSpinner("Summoning Expo CLI", "🪔")
       stopSpinner("Cleaning up Expo install", "🎫")
     } else {
+      // remove pods and node_modules, if they exist, because those will be rebuilt anyway
+      startSpinner("Igniting app")
+      remove(path(boilerplatePath, "ios", "Pods"))
+      remove(path(boilerplatePath, "node_modules"))
+      stopSpinner("Igniting app", "🔥")
+
+      startSpinner(" 3D-printing a new React Native app")
+      await copyBoilerplate(toolbox, {
+        boilerplatePath,
+        projectName,
+        excluded: ["node_modules", "yarn.lock", /.?\.expo\..?/],
```

Comment:
nice! Excluding all expo files with a regex :D

Reply:
Took me a bit to remember if the wildcard should be `?.` or `.?`. haha